### PR TITLE
[WIP] Dynamic metadata for challenges

### PR DESCRIPTION
**Pull request by gautamjajoo**
Working on it.
Currently, facing the error: `error Parsing error: 'import' and 'export' may appear only with 'sourceType: module'`

### CODE REVIEW

File: frontend/src/js/route-config/route-config.js [hunk 1]
```diff
@@ -30,7 +33,10 @@
             templateUrl: baseUrl + "/web/landing.html",
             controller: 'MainCtrl',
             controllerAs: 'main',
-            title: "Welcome"
+            meta: {
+                  'title': 'Welcome',
+                  'url':
```
Comment (Ram81) [inline frontend/src/js/route-config/route-config.js:38]: @gautamjajoo `url` key needs a value here.

File: frontend/src/js/route-config/route-config.js [hunk 2]
```diff
@@ -589,8 +597,6 @@
         });
 
         $rootScope.$on('$stateChangeSuccess', function() {
-            // Save the route title
-            $rootScope.pageTitle = $state.current.title;
```
Comment (Ram81) [inline frontend/src/js/route-config/route-config.js:593]: why are we removing this?

**Review by Ram81 — CHANGES_REQUESTED** [review]

**Comment by Ram81** [comment]
@gautamjajoo can you try the fixes mentioned [here](https://github.com/AtomLinter/linter-eslint/issues/462) for the error you are facing

File: frontend/src/js/route-config/route-config.js (hunk 2 above)
Comment (gautamjajoo) [reply frontend/src/js/route-config/route-config.js:593]: ngMeta also updates the title of the page

**Pull request closed without merging**